### PR TITLE
kubeadm: upgrade ha from 1.13 to 1.14

### PR DESCRIPTION
**Pull request by yagonobre**
Place holder for kubeadm upgrade HA of [kubernetes/kubeadm/issues/1422](https://github.com/kubernetes/kubeadm/issues/1422)

/assign @neolit123

### CODE REVIEW

**Comment by zacharysarah** [comment]
@yagonobre I changed the base of this PR to `dev-1.14` instead of `release-1.4`.

**Comment by yagonobre** [comment]
Thanks @zacharysarah, I'll update in a bit

**Comment by neolit123** [comment]
@yagonobre just nuke your local branch and do:

```
git fetch <upstream-remote> dev-1.14:kubeadm-doc
```

the checkout that new branch add the change and force push.

**Comment by yagonobre** [comment]
Thanks @neolit123 I was trying to rebase without success. 
/remove-language zh

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha-1-14.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+Check prerequisites and determine the upgrade versions:
+
+```shell
+kubeadm upgrade plan
+```
+
+You should see something like the following:
+
+```
+Upgrade to the latest version in the v1.13 series:
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-ha-1-14.md]: 1.14.

**Comment by neolit123** [comment]
i will close this PR and send a new one because we are doing pair review with @fabriziopandini currently.

**Comment by neolit123** [comment]
closing because we are going to merge HA upgrade with regular upgrade.
/close

**Pull request closed without merging**